docs/documentation/development/conventions.md

```diff
@@ -18,11 +18,24 @@ For private methods or methods in an internal package the annotations are respec
 
 To use the annotations, every bundle must have an **optional** `Import-Package` dependency to `org.eclipse.jdt.annotation`.
 Classes should be annotated by `@NonNullByDefault` and return types, parameter types, generic types etc. are annotated with `@Nullable` only.
+The annotation should be written before the type, not before the visibility modifier.
```

Comment:
I propose to change "before" to "in front of".

Reply:
Should this only be done for field declarations (like, e.g., `private @Nullable String bla;`)? Or also on the class level (like, e.g., `public @NonNullByDefault class Blubb`)?

Reply:
I agree